### ISSUE TITLE
Bug with LARAVEL_CODE_QUALITY set list

### ISSUE THREAD

**Issue by rcerljenko** [driftingly/rector-laravel#278]
Hi,

Since update to v2, this happens:
```
[ERROR] syntax error, unexpected token "new", expecting "]"
```

And I noticed that it only happens when including `LARAVEL_CODE_QUALITY` set list.
Inspecting the rules that this activates I noticed that this rule doesn't exist anymore but config is trying to load it:
```php
use RectorLaravel\Rector\Coalesce\ApplyDefaultInsteadOfNullCoalesceRector;
```

Line 14:
![image](https://github.com/user-attachments/assets/4e15714f-6ce4-444b-ae46-cf1ccce7f341)

**Comment by GeniJaho** [comment]
This is the weirdest issue 😅 
I noticed it as well, in the downloaded code in `vendor` there were some commas missing, breaking the code and causing the cryptic error.
It has been fixed in core Rector tags ([2.0.1](https://github.com/rectorphp/rector/commit/5e3504c4c9f2c668a01bc957b67125047b753cd2), [2.0.2](https://github.com/rectorphp/rector/commit/8f1c86681e6abba5f1cd790b18539b2c94008a6e), [2.0.3](https://github.com/rectorphp/rector/commit/3f27091368bd935dbbaa8387099792fb20f65f68)), but not before we did our own release here, we should have waited a bit more 😅 .

I triggered another Downgrade Release for Rector Laravel, and it seems to work for me. Can you pull the 2.0 release again please, @rcerljenko ?

**Comment by rcerljenko** [comment]
It works thanks!  :)
